collections/_posts/2022-11-02-NeighborhoodAttentionTransformer.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: review
+title: "Neighborhood Attention Transformer"
+tags: deep-learning CNN transformer segmentation classification object-detection attention
+author: "Pierre Rougé"
+cite:
+    authors: "Ali Hassani, Steven Walton, Jiachen Li, Shen Li, Humphrey Shi"
+    title:   "Neighborhood Attention Transformer"
+pdf: "https://arxiv.org/abs/2204.07143v1"
+---
```

Comment:
The venue where the paper was published is missing. If the paper has only been published on arXiv as of yet, put arXiv as venue. That way, it's clear the paper is available as a pre-print and it's not just an oversight.